lib/src/auth/user/user_info.dart

```diff
@@ -17,29 +17,30 @@ class UserInfo {
   factory UserInfo.fromMap(Map<String, dynamic> map) {
     final List<String> groups = [];
     if (map.containsKey('group')) {
-      groups.add(map['group']);
-    }
-    if (map.containsKey('groups') && groups.isEmpty) {
-      final serializedGroups = map['groups'] as List<dynamic>;
-      groups.addAll(serializedGroups.cast<String>());
+      final serializedGroup = map['group'];
+      if(serializedGroup is List) {
+        groups.addAll(serializedGroup.cast<String>());
+      } else {
+        groups.add(serializedGroup);
+      }
     }
     if(groups.isEmpty) {
```

Comment:
may be we can remove this conditional throw.
if groups is empty it means user not applied to the any group, but such user exists and valid.
the question is... how will work application code which checks for user having necessary group to access?
possible we have to fix some parts of application code to reach proper group validation if user has empty list of groups.